views/jira-configuration.hbs

```diff
@@ -305,6 +306,16 @@
               organization to connect and start including issue keys in
               branches, commit messages or pull requests to see development
               insights in Jira.</p>
+            <button
+              class="ak-button ak-button__appearance-default learn-more-link"
```

Comment:
We shouldn't be using a button here as using JS to redirect to another page isn't supported by screen readers. We could add an aria role but it's always better to use native elements where possible as they are more widely supported by assistive technologies.

- [ ] Change the button to a link
- [ ] ditch the JS 
- [ ] style the link to look like a button

Reply:
- Button modified to `<a>` tag using the same class name, maintaining the CSS.
- JS function removed